changelog/1374.doc.rst

```diff
@@ -1 +1 @@
-Moved references from individual docstrings to the bibliography.
+Moved references from individual docstrings to the |bibliography|.
```

Comment:
Also not not rendering correctly on RTD.

![image](https://user-images.githubusercontent.com/29869348/155811153-9efed39b-ba11-4004-8cbb-0c26839a50fd.png)